install-i3lock-color.sh

```diff
@@ -6,3 +6,11 @@ cd build
 sudo make install
 
 sudo install -Dm644 ../LICENSE "/usr/share/licenses/i3lock-color/LICENSE"
+
```

Comment:
I don't actually know how all the make process works here, so I made this workaround